.github/workflows/integration-test.yml

```
@@ -1,137 +1,60 @@
 ---
 name: Integration Test
-run-name: |
-  Integration Test on ${{ inputs.test-platform }}-${{ inputs.test-architecture }} (${{ inputs.test-run == 'all' && 'all' || format('{0}={1}', inputs.test-run, inputs.test-parameter) }})
-
-env:
-  DEFAULT_TEST_PLATFORM: kind-1.31.0
-  DEFAULT_TEST_ARCHITECTURE: amd64
-  DEFAULT_TEST_RUN: all
-  DEFAULT_TEST_PARAMETER: "" # Unused when the test-run is 'all'
-  TEST_PLATFORM: ${{ inputs.test-platform }}
-  TEST_ARCHITECTURE: ${{ inputs.test-architecture }}
-  TEST_RUN: ${{ inputs.test-run }}
-  TEST_PARAMETER: ${{ inputs.test-parameter }}
 
 on:
   # schedule:
     # At 00:00 on Sunday. See: https://crontab.guru/#0_0_*_*_0
     # - cron: "0 0 * * 0"
```

This should be indented like this:

```suggestion
  # schedule:
  #   # At 00:00 on Sunday. See: https://crontab.guru/#0_0_*_*_0
  #   - cron: "0 0 * * 0"
```

---

Can be ignored for now, and done in a follow up templating run.

---

I have the necessary local changes ready to go in after https://github.com/stackabletech/operator-templating/pull/546.